src/client/java/teammates/client/scripts/sql/DataMigrationForCourseEntitySql.java

```diff
@@ -1,59 +1,97 @@
 package teammates.client.scripts.sql;
 
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
 import java.time.Instant;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.Collectors;
 
+import com.google.cloud.datastore.Cursor;
+import com.google.cloud.datastore.QueryResults;
 import com.googlecode.objectify.cmd.Query;
 
+import teammates.client.connector.DatastoreClient;
+import teammates.client.util.ClientProperties;
+import teammates.common.util.Const;
+import teammates.common.util.HibernateUtil;
 import teammates.storage.entity.Course;
 import teammates.storage.entity.CourseStudent;
+import teammates.storage.sqlentity.BaseEntity;
 import teammates.storage.sqlentity.Section;
 import teammates.storage.sqlentity.Student;
 import teammates.storage.sqlentity.Team;
+import teammates.test.FileHelper;
 
 /**
  * Data migration class for course entity.
  */
 @SuppressWarnings("PMD")
-public class DataMigrationForCourseEntitySql extends
-        DataMigrationEntitiesBaseScriptSql<teammates.storage.entity.Course, teammates.storage.sqlentity.BaseEntity> {
+public class DataMigrationForCourseEntitySql extends DatastoreClient {
+
+    private static final String BASE_LOG_URI = "src/client/java/teammates/client/scripts/log/";
+
+    private static final int BATCH_SIZE = 1000;
```

Comment:
Should we also decrease the `BATCH_SIZE`, since querying for courses itself is not the bottleneck

Reply:
Reduced to 100 for now.